connectors/services/__init__.py

```diff
@@ -3,7 +3,10 @@
 # or more contributor license agreements. Licensed under the Elastic License 2.0;
 # you may not use this file except in compliance with the Elastic License 2.0.
 #
+# fmt: off
```

Comment:
Why tho?

Reply:
I think it's a sign our names for classes are too long, can we shorten them maybe?

Reply:
Oh wow, why is it removed?

Reply:
Now I understand why.

`make autoformat` will run `black` first, then `ruff`:
https://github.com/elastic/connectors/blob/d1d7f9a72ab6cdc5cefc0d65c05390a9a5b39bd3/Makefile#L45-L53

`black` will change
```python
from connectors.services.access_control_sync_job_execution import (
    AccessControlSyncJobExecutionService,  # NOQA
)
```

to
```python
from connectors.services.access_control_sync_job_execution import (
    AccessControlSyncJobExecutionService, 
)  # NOQA
```
Because `# NOQA` is not applied at the 2nd line, and `AccessControlSyncJobExecutionService` is imported but not used, `ruff` will remove it.

Reply:
https://github.com/elastic/connectors/pull/1925/commits/a435d797c1c6ed303628e6e961c7e07879580792 should make it work

Reply:
Note to self: we can probably get rid of `black` as `ruff` is already doing its job